src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -107,8 +107,9 @@ public int getTotalMessageCount(){
         String user = (String) entity.getProperty("user");
         String text = (String) entity.getProperty("text");
         long timestamp = (long) entity.getProperty("timestamp");
+        String recipient = (String) entity.getProperty("recipient");
 
-        Message message = new Message(id, user, text, timestamp);
+        Message message = new Message(id, user, text, timestamp, recipient); //this line was causing an error
```

Comment:
//this line was causing an error
What is this comment for?

Reply:
I was trying to explain why I changed it. I know Joel added these lines but I needed to change them to work for my step 5.